beacon-chain/p2p/subnets.go

```diff
@@ -429,6 +431,25 @@ func (s *Service) updateSubnetRecordWithMetadataV3(
 		Syncnets:          bitVSync,
 		CustodyGroupCount: custodyGroupCount,
 	})
+	s.saveMetaDataIfNeeded()
+}
+
+// saveMetaDataIfNeeded saves the metadata to the file if the static peer ID flag is set.
+func (s *Service) saveMetaDataIfNeeded() {
+	if s.cfg.StaticPeerID {
```

Comment:
I think this will also need to be updated to if (s.cfg.StaticPeerID || params.FuluEnabled()) I think we persist it post fulu 

@nalepae to reconfirm

Reply:
Did we consider writing this to the database instead of adding a new separate file?